frontend/src/components/messstelle/optionsmenue/OptionsmenueMessstelle.vue

```diff
@@ -1,232 +1,247 @@
+import { useDisplay } from "vuetify";
+
+import DarstellungsoptionenPanelMessstelle from "@/components/messstelle/optionsmenue/panels/DarstellungsoptionenPanelMessstelle.vue";
 import FahrzeugPanel from "@/components/messstelle/optionsmenue/panels/FahrzeugPanelMessstelle.vue";
-import DefaultObjectCreator from "@/util/DefaultObjectCreator";
-import MessstelleInfoDTO from "@/types/messstelle/MessstelleInfoDTO";
+import MessquerschnittPanel from "@/components/messstelle/optionsmenue/panels/MessquerschnittPanel.vue";
+import ZeitPanel from "@/components/messstelle/optionsmenue/panels/ZeitPanel.vue";
+import { useMessstelleStore } from "@/store/messstelle";
+import { useSnackbarStore } from "@/store/snackbar";
+import { useUserStore } from "@/store/user";
 import DetektierteFahrzeugart from "@/types/enum/DetektierteFahrzeugart";
-import _ from "lodash";
+import TagesTyp from "@/types/enum/TagesTyp";
 import ZaehldatenIntervall from "@/types/enum/ZaehldatenIntervall";
-import Zeitblock from "@/types/enum/Zeitblock";
 import Zeitauswahl from "@/types/enum/Zeitauswahl";
-import MessquerschnittPanel from "@/components/messstelle/optionsmenue/panels/MessquerschnittPanel.vue";
+import Zeitblock from "@/types/enum/Zeitblock";
+import DefaultObjectCreator from "@/util/DefaultObjectCreator";
 import { useMessstelleUtils } from "@/util/MessstelleUtils";
-import TagesTyp from "@/types/enum/TagesTyp";
-import DarstellungsoptionenPanelMessstelle from "@/components/messstelle/optionsmenue/panels/DarstellungsoptionenPanelMessstelle.vue";
-import { useSnackbarStore } from "@/store/snackbar";
-import { useMessstelleStore } from "@/store/messstelle";
-import { useUserStore } from "@/store/user";
 import { useTimeUtils } from "@/util/TimeUtils";
+import {useDateUtils} from "@/util/DateUtils";
 
 interface Props {
-    messstelleId: string;
+  messstelleId: string;
 }
 
 defineProps<Props>();
 
-const vuetify = useVuetify();
+const display = useDisplay();
 const messstelleStore = useMessstelleStore();
 const snackbarStore = useSnackbarStore();
 const messstelleUtils = useMessstelleUtils();
 const dialog = ref(false);
 const chosenOptions = ref(
-    DefaultObjectCreator.createDefaultMessstelleOptions()
+  DefaultObjectCreator.createDefaultMessstelleOptions()
 );
 
 const userStore = useUserStore();
 const timeUtils = useTimeUtils();
+const dateUtils = useDateUtils();
 
-const messstelle: Ref<MessstelleInfoDTO> = computed(() => {
-    return messstelleStore.getMessstelleInfo;
+const messstelle = computed<MessstelleInfoDTO>(() => {
+  return messstelleStore.getMessstelleInfo;
 });
 
 const getContentSheetHeight = computed(() => {
-    if (vuetify.breakpoint.xl) {
-        return "650px";
-    }
-    return "400px";
+  if (display.xl.value) {
+    return "650px";
+  }
+  return "400px";
+});
+
+const buttonText = computed(() => {
+  let text = "Filtereinstellungen";
+  if (display.xl.value) {
+    text = "Filtereinstellungen bearbeiten";
```

Comment:
Weshalb wird der Text des Buttons entsprechend der größe Unterschieden. Der Informationsgehalt in den Texten ist in der selbe.

Vorschlag: Reduzieren auf "Filtersintellungen". Somit würde dann der Code entfallen und verschlankt werden.